package.json

```diff
@@ -41,14 +41,14 @@
     "@chainsafe/eth2.0-spec-test-util": "^0.3.0",
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^10.12.17",
+    "@types/node": "^12.0.10",
```

Comment:
@GregTheGreek Woah, can we prevent it from updating major changes, as those usually have breaking changes? 
I marked this because I'm pretty sure @types/node major version is aligned with nodejs version

Reply:
Looking into it